src/extension/display.js

```diff
@@ -30,6 +30,14 @@ export async function setDisplay(display) {
   await setConfig('local', {
     display,
   });
+
+  const mod = await import(chrome.runtime.getURL('utils/rum.js'));
+  const { default: sampleRUM } = mod;
+
+  sampleRUM('click', {
+    source: 'sidekick',
+    target: `${display ? 'shown' : 'hidden'}`,
```

Comment:
Move these to `EXTERNAL_EVENTS`?

Reply:
Just a clarification. By external events do we mean events dispatched by the custom element? Currently only `hidden` is dispatched from the element.

Reply:
Right, I guess I got events and checkpoints confused.